app.coffee

```diff
@@ -147,6 +152,7 @@ app.io.route 'chat:msg', (req)->
 app.io.route 'chat:demand', (req)->
   logs = messages.toArray()
   req.io.emit 'chat:log', logs
+  slack.postMessage req.data.nick + " entered channel", process.env.SLACK_CHANNEL, "Jinora"
```

Comment:
We should also send a message when the user is disconnected.
Also we might want to make this optional since this will send a notification to everyone whenever a user enters #public channel and many will not like it. I would like to hear others' views here.